docs/plugins.rst

```diff
@@ -50,3 +53,32 @@ where `myproject.pluginmodule` points to a Renku `hookimpl` e.g.:
 
 .. automodule:: renku.core.plugins.run
    :members:
+
+
+CLI Plugins
+-----------
+
+Command-line interface plugins are supported using the `click-plugins <https://github.com/click-contrib/click-plugins>` library.
+
+As in case the runtime plugins, command-line plugins can be created as python packages that contain the respective entrypoint definition in their `setup.py` file, like so:
```

Comment:
```suggestion
As in case the runtime plugins, command-line plugins can be created as Python packages that contain the respective entry point definition in their `setup.py` file, like so:
```